server_installation/topics/installation/distribution-files-product.adoc

```diff
@@ -1,9 +1,25 @@
 
 === Installing RH-SSO from a ZIP File
 
-The {project_name} Server is contained in one distribution file: rh-sso-{project_version}.zip.gz.
+The {project_name} server download ZIP file contains the scripts and binaries to run the {project_name} server. You install the 7.4.0 server first, then the {project_version} server patch.
+
+.Procedure
+
+. Go to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[Red Hat customer portal].
+
+. Download the {project_name} Server: *rh-sso-7.4.0.zip*
+
+. Unpack the ZIP file using the appropriate `unzip` utility, such as unzip, tar, or Expand-Archive.
+
+. Return to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=core.service.rhsso[Red Hat customer portal].
+
+. Click the `Patches` tab.
+
+. Download the server patch: *rh-sso-{project_version}.zip*
+
+. Place the file in a directory you choose.
+
+. Unpack the ZIP file.
```

Comment:
@andymunro Also here, I do not understand why the latest installation manual was changed into patch installation? Normally the user is looking for a first time installation. The patch installation should have its own chapter.